Example/src/TagInputExample.js

```diff
@@ -39,6 +59,23 @@ export default class TagInputExample extends Component {
             maxHeight={75}
           />
         </View>
+
+        <Text style={{marginVertical: 10}}>Horizontal Scroll</Text>
+        <View style={{marginBottom: 10, flexDirection: 'row', alignItems: 'center', backgroundColor: 'lightblue'}}>
```

Comment:
The way we generally use the example project is to just iterate through all the properties to test them individually. We'd have a whole lot of examples here if we just created a new one for every property we have. I think we should just keep this file as-is.

Reply:
Actually, I thought about this more, and in retrospect I think it makes sense to have this separate example, considering that it's considerably more complex than just setting a couple properties.